src/page.ts

```diff
@@ -253,27 +291,15 @@ export class Page {
     options?: ScreenshotOptions,
   ): Promise<string> {
     if (!existsSync(path)) {
-      await this.#protocol.done();
-      throw new Error(`The provided folder path - ${path} doesn't exist`);
+      await this.client.close(
+        `The provided folder path - ${path} doesn't exist`,
```

Comment:
suggesting that we wrap the path in quotes as opposed to using a hyphen to denote exactly what the path is that doesn't exist

```suggestion
        `The provided folder path "${path}" doesn't exist`,
```